### PR TITLE
MDN Feature Pages for SVGAnimatedRect: animVal, baseVal

### PR DESCRIPTION
**Pull request by yashrajbharti**
part of [Missing baseline pages](https://github.com/openwebdocs/project/issues/214) project

### CODE REVIEW

File: files/en-us/web/api/svganimatedrect/animval/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+<script>
+  const svgElement = document.getElementById("mySvg");
+  const animatedRect = svgElement.viewBox.animVal;
+
+  // Log the animated value (assuming an animation is applied)
+  console.log(animatedRect.x);
+  console.log(animatedRect.y);
+  console.log(animatedRect.width);
+  console.log(animatedRect.height);
+</script>
```
Comment (estelle) [inline files/en-us/web/api/svganimatedrect/animval/index.md:35]:
```suggestion
```js
  const svgElement = document.getElementById("mySvg");
  const animatedRect = svgElement.viewBox.animVal;

  // Log the animated value (assuming an animation is applied)
  console.log(animatedRect.x);
  console.log(animatedRect.y);
  console.log(animatedRect.width);
  console.log(animatedRect.height);
```

File: files/en-us/web/api/svganimatedrect/animval/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+## Value
+
+A {{domxref("DOMRectReadOnly")}} object representing the animated value of the `viewBox` attribute.
+
+## Examples
+
+```html
+<svg viewBox="0 0 200 100" id="mySvg">
+  <rect width="100" height="100" fill="blue" />
+</svg>
```
Comment (estelle) [inline files/en-us/web/api/svganimatedrect/animval/index.md:24]: add a new line with three tick marks \` to end the html. 
In examples, don't use `<script>` or `<style>`. Rather, separate any html, css, and js into separate code blocks.

File: files/en-us/web/api/svganimatedrect/animval/index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+slug: Web/API/SVGAnimatedRect/animVal
+page-type: web-api-instance-property
+browser-compat: api.SVGAnimatedRect.animVal
+---
+
+{{APIRef("SVG")}}
+
+The **`animVal`** read-only property of the {{domxref("SVGAnimatedRect")}} interface represents the current animated value of the `viewBox` attribute of an SVG element.
+
+This property reflects the SVG element's {{SVGAttr("viewBox")}} attribute value as a read-only {{domxref("DOMRectReadOnly")}} object. If the `viewBox` attribute is being animated, animVal provides access to the rectangle's dynamic state, including the x, y, width, and height values during the animation. If no animation is applied, `animVal` will be identical to {{domxref("SVGAnimatedRect.baseVal")}}.
```
Comment (estelle) [inline files/en-us/web/api/svganimatedrect/animval/index.md:13]:
```suggestion
The **`animVal`** read-only property of the {{domxref("SVGAnimatedRect")}} interface represents the current animated value of the `viewBox` attribute of an SVG element as a read-only {{domxref("DOMRectReadOnly")}} object. It provides access to the rectangle's dynamic state, including the `x`, `y`, `width`, and `height` values during the animation.

If no animation is applied, the `animVal` property reflects the SVG element's {{SVGAttr("viewBox")}} attribute value and will be identical to {{domxref("SVGAnimatedRect.baseVal")}}.
```
add tic marcs around text that is code, such as the values and and the property name.
Suggested sentence re-order to do what it is and the type it returns in the first paragraph, and reflection and equivalents in the last paragraph.

File: files/en-us/web/api/svganimatedrect/baseval/index.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+slug: Web/API/SVGAnimatedRect/baseVal
+page-type: web-api-instance-property
+browser-compat: api.SVGAnimatedRect.baseVal
+---
+
+{{APIRef("SVG")}}
+
+The **`baseVal`** read-only property of the {{domxref("SVGAnimatedRect")}} interface represents the current non-animated value of the `viewBox` attribute of an SVG element.
+
+This property reflects the SVG element's {{SVGAttr("viewBox")}} attribute value as a read-only {{domxref("DOMRect")}} object. It provides access to the static rectangle defined by the `viewBox` attribute, including the x, y, width, and height values.
```
Comment (estelle) [inline files/en-us/web/api/svganimatedrect/baseval/index.md:13]:
```suggestion
The **`baseVal`** read-only property of the {{domxref("SVGAnimatedRect")}} interface represents the current non-animated value of the `viewBox` attribute of an SVG element.

This property reflects the SVG element's {{SVGAttr("viewBox")}} attribute value as a read-only {{domxref("DOMRect")}} object. It provides access to the static rectangle defined by the `viewBox` attribute, including the `x`, `y`, `width`, and `height` values.
```

File: files/en-us/web/api/svganimatedrect/baseval/index.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+<script>
+  const svgElement = document.getElementById("mySvg");
+  const animatedRect = svgElement.viewBox.baseVal;
+
+  // Access the non-animated base value
+  console.log(animatedRect.x); // 0
+  console.log(animatedRect.y); // 0
+  console.log(animatedRect.width); // 200
+  console.log(animatedRect.height); // 100
+</script>
```
Comment (estelle) [inline files/en-us/web/api/svganimatedrect/baseval/index.md:35]: same comments as above

**Review by estelle — COMMENTED** [review]
Thanks. a few suggestions

**Comment by yashrajbharti** [comment]
All fixed!

**Review by estelle — APPROVED** [review]
Awesome. Thank you! 🎉

**Pull request merged**